ci/build.sh

```diff
@@ -2,11 +2,18 @@
 
 set -ex
 
-COMPILER="$1"
-RELEASE="$2"
-ABI="$3"
-BLOSC="$4"
-SIMD="$5"
+COMPILER="$1"; shift
+RELEASE="$1"; shift
+ABI="$1"; shift
+BLOSC="$1"; shift
+SIMD="$1"; shift
+CMAKE_EXTRA="$@"
```

Comment:
Changed this slightly to allow for arbitrary CMake commands after this positional args